### DISCUSSION TITLE
Alpinejs slowing down Plotly.js chart

### DISCUSSION THREAD

**Discussion by Tesla2k** [alpinejs/alpine#1812] [1. Help]
Hello, 
I have a chart made with plotly, without Alpine it runs ok, but when I include alpine the performance drops down to unusable, when dragging or zooming. I first thought updating from v2 to v3 fixed the issue, but then I realized I have forgotten to call Alpine.start(), now the performance is as bad as before.

I tried to put everything in a div with x-ignore but it did not help.

Any suggestions?

**Comment by pfeatherstone** [comment]
Could you post some code and submit an issue?

**Comment by Goblin80** [comment]
> By default, Alpine will crawl and initialize the entire DOM tree of an element containing x-init or x-data.

consider adding [x-ignore](https://alpinejs.dev/directives/ignore) to plotly's chart div.
